### PR TITLE
Document test setup.

### PR DESCRIPTION
**Pull request by khink**
Document how to setup a virtualenv for running test, for dummies.

### CODE REVIEW

File: docs/source/internals/contributing/running-tests.rst [hunk 1]
```diff
@@ -7,6 +7,8 @@ Running tests
 
 Oscar uses pytest_ to run the tests, which can be invoked using::
 
+    $ mkvirtualenv oscar-tests
+    $ pip install -e .[test]
```
Comment (sasha0) [inline docs/source/internals/contributing/running-tests.rst]: I think we should add a sentence, saying that next commands allows to create virtual environment and install dependencies in order to run tests.

**Comment by khink** [comment]
There seems to be more, i get test failures: https://gist.github.com/khink/0973018b149d3a985393d9d43a5ae291

**Comment by khink** [comment]
Looks like it does work with python2.7.  I used python3.4 (3.4.7 actually), will try with 3.6 and update.

**Comment by khink** [comment]
I see a "bad magic number" which suggests i should throw away `.pyc` files. That fixed it, indeed.

I'm ready to start writing tests now, and i hope this addition will help others getting started more quickly.

**Comment by sasha0** [comment]
Good good.

Thanks for the contribution!

**Comment by khink** [comment]
Thanks for your speedy response, and your help!

**Comment by sasha0** [comment]
Np!

**Pull request merged**